string table
string key
string value
---
bool status
